README.md

```diff
@@ -232,17 +232,29 @@ cd $GOPATH/src/github.com/kubeedge/kubeedge
 
 ### Build Cloud
 
-```shell
-cd $GOPATH/src/github.com/kubeedge/kubeedge/cloud/edgecontroller
-make # or `make edgecontroller`
-```
++ Modify the `$GOPATH/src/github.com/kubeedge/kubeedge/cloud/edgecontroller/conf/ontroller.yaml` configuration file to modify `cloudhub.ca`, `cloudhub.cert`, `cloudhub.key` to generated Certificate path
```

Comment:
It is better to move the modification of certifacte into `Run Cloud` and `Run Edge`
What do you think about that?

Reply:
I think it depends on whether I need to re-make after modifying the `conf/*.yaml` file.Do I need to re-make?

Reply:
@zhanghongtong no need to re-make after modifying the conf/*.yaml file:)

Reply:
OK :）